shard.go

```diff
@@ -76,6 +76,28 @@ func (s *cacheShard) set(key string, hashedKey uint64, entry []byte) error {
 	}
 }
 
+func (s *cacheShard) del(key string, hashedKey uint64) error {
+	s.lock.RLock()
+	itemIndex := s.hashmap[hashedKey]
+
+	if itemIndex == 0 {
+		s.lock.RUnlock()
+		return notFound(key)
+	}
+
+	wrappedEntry, err := s.entries.Get(int(itemIndex))
+	if err != nil {
+		s.lock.RUnlock()
+		return err
```

Comment:
Please, add a test for this case.

Reply:
Sorry, it's really hard to test this case.

There're three error cases for `queue.BytesQueue#Get`:

1. `itemIndex <= 0`: 0 already covered and `uint` never negative.
2. empty queue & 3 index size overflow have no way to be faked.

Actually, the [same logic](https://coveralls.io/builds/13894823/source?filename=shard.go#L36) in `bigcache#Get` is also not unittest covered.

Any suggestions?